subprojects/docs/src/docs/release/notes.md

```diff
@@ -29,11 +29,14 @@ There are a number of limitations to be aware of. These will be improved in late
 
 Caching has always been one of the strong suits of Gradle. Over time, more and more persistent caches have been added to improve performance and support new features, requiring more and more disk space on build servers and developer workstations. Gradle now addresses one of the most highly voted issues on GitHub and introduces the following cleanup strategies:
 
-- Version-specific cache directories in `GRADLE_USER_HOME/caches/<gradle-version>/` are checked periodically (at most every 24 hours) for whether they are still in use. If not, directories for release versions are deleted after 30 days of inactivity, snapshot versions after 7 days of inactivity. Moreover, the corresponding Gradle distributions in `GRADLE_USER_HOME/wrapper/dists/` are deleted as well, if present.
-- Similarly, after building a project, version-specific cache directories in `PROJECT_DIR/.gradle/<gradle-version>/` are checked periodically (at most every 24 hours) for whether they are still in use. They are deleted if they haven't been used for 7 days.
+- Version-specific cache directories in `GRADLE_USER_HOME/caches/<gradle-version>/` are checked periodically (at most every 24 hours) for whether they are still in use. If not, directories for release versions are deleted after 30 days of inactivity, snapshot versions after 7 days of inactivity.
+- Gradle distributions in `GRADLE_USER_HOME/wrapper/dists/` are checked periodically (at most every 24 hours) for whether they are still in use, i.e. whether there's a corresponding version-specific cache directory. Unused distributions are deleted.
```

Comment:
I still feel like these details should be in the user guide, not in the release notes. If we don't have a good home for this kind of information yet, maybe we should create one? If I search for "Gradle user home" in our documentation, I find nothing that explains to me what it is, what it contains and how it's cleaned up. Same for the `.gradle` directory in the project.

Reply:
@eriwen @pledbrook Do you know where to put this? Maybe the Build Environment chapter?

Reply:
The title Build Environment seems kind of close to this, but currently is positioned for Configuring the Build Environment. This doesn't _quite_ fit, but we can change that.

What I think might be better would be an overview of Gradle-managed directories. What about "Gradle Directory Layout" similar to what [Bazel has](https://docs.bazel.build/versions/master/output_directories.html)? 

I would hope such a document would explain:

 * These top-level directories under `GRADLE_USER_HOME/`: caches, daemon, init.d, and wrapper
   * where to find Daemon logs
   * Where to put init scripts and gradle.properties
   * where dependency JARs get stored, roughly
 * What roughly goes under `PROJECT_DIR/.gradle/<version>`

Do you agree folks?

Reply:
Yes I think that would be useful.

Reply:
Either that, or put this information in the chapter that deals with the feature(s) that rely on those directories. For example, the `dists` directory can be discussed in the Gradle Wrapper chapter. I don't know what the version-specific cache directory is for. Several things it appears.

Since some directories and files don't seem to be tied to a particular Gradle feature, it would be good to have something like an appendix on the directories and files that Gradle creates and consumes. Other chapters could link to it where appropriate.

Reply:
Thanks for the feedback! I'll take a stab at a "Gradle Directory Layout" chapter in the next few days.